tests/templates.test.js

```diff
@@ -92,14 +92,13 @@ describe("Test JSON validation scenarios", () => {
 
     client.getSession.mockReturnValueOnce([mockSession, false]);
     const { req, res } = createMocks({
-      method: "POST",
+      method: "INSERT",
```

Comment:
I don't think this is correct. This is the method of the mocked HTTP request

Reply:
Good catch.

Reply:
This was a cross change that got captured in a stash.  Reverting